_posts/2023-03-16-accelerated-diffusers-pt-20.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: blog_detail
 title: "Accelerated Diffusers with PyTorch 2.0"
-author: Pedro Cuenca, Patrick von Platen, Suraj Patil
+author: Pedro Cuenca, Patrick von Platen, Suraj Patil, Sayak Paul
```

Comment:
Sayak created the new benchmark script and docker images, ran the benchmark on various GPUs for several pipelines and collated results.